### ISSUE TITLE
[Adaptive Cards> FlightDetails.Json]: Luminosity ratio of 'Flight to JFK text is 3.269:1 which is less than 4.5:1.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8631]
### Target Platforms

UWP

### SDK Version

1.2.2306.30005

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 23H2 (Build 25905.1000)
Application: AdaptiveCardVisualizer
Application Version: 1.2.2306.30005
Tool: Color Contrast Analyzer through Accessibility Insights for Windows

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to the 'FlightDetails.json' through tab key and activate it.
3. Navigate to 'Flight to JFK' text.
4. Pick the color for foreground and background of 'Flight to JFK' text.
5. Observe the issue.

**Actual Result:**
Luminosity ratio of 'Flight to JFK text is 3.269:1 which is less than 4.5:1.

**Expected Result:**
Luminosity ratio of 'Flight to JFK text should be greater than or equals to 4.5:1.

**User Impact:**
Person with visual disability will get impacted as user will not be able to see the text properly and will not be able to understand the purpose of the text.

**Reference link:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “
<img width="960" alt="MAS1 4 3 Luminosity ratio of Flight to JFK text is less than required" src="https://github.com/microsoft/AdaptiveCards/assets/104483005/778fa729-0cbb-4b8f-9e74-e026ed5c46a7">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @jwoo-msft,

Could you please look in this issue and provide us ETA  to fix this issue as of now due to this issue Product is getting non-compliance.

**Comment by vagpt** [comment]
This issue is working fine, hence closing this issue.

![image](https://github.com/user-attachments/assets/b654dd3c-9114-48f4-ba6d-9ca9a94cfb4d)
